wiki/Gameplay/Game_modifier/Double_Time/de.md

```diff
@@ -0,0 +1,62 @@
+---
+tags:
+  - doubletime
+  - double-time
+  - DT
+  - mod
+  - game modifier
+  - Spielmodifikation
+---
+
+# Double Time (Mod)
+
+![DT Modsymbol](/wiki/shared/mods/DT.png "Double Time (DT) Modsymbol")
+
+*Für die vollständige Liste aller Mods, siehe: [Spielmodifikationen](/wiki/Game_modifier)*\
+*Siehe auch: [Nightcore (Mod)](/wiki/Game_modifier/Nightcore)*
+
+## Übersicht
+
+- Abkürzung: DT
+- Typ: Erhöhung der Schwierigkeit
+- Score-Multiplikator:
+  - ![][osu!]: 1,12x
+  - ![][osu!taiko]: 1,12x
+  - ![][osu!catch]: 1,06x
+  - ![][osu!mania]: 1,00x
+- Standard-Tastenkürzel: `D`
+- Untertitel: `Brumm brumm`
+- Kompatible Spielmodi: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Variante: [Nightcore](/wiki/Game_modifier/Nightcore)
+
+## Beschreibung
+
+*Anmerkung: Die Methode zur Erhöhung der Geschwindigkeit führt nicht zu einer Erhöhung der Tonhöhe des Songs.*
+
+Die Mod **Double Time** ist eine [Spielmodifikation](/wiki/Game_modifier), die versucht, die Schwierigkeit einer [Beatmap](/wiki/Beatmap) zu erhöhen, indem die allgemeine Geschwindigkeit (BPM) um 150 % (1,5x) erhöht wird, die Länge des Songs um 33 % reduziert wird und die [Approach-Rate (AR)](/wiki/Beatmapping/Approach_rate), [allgemeine Schwierigkeit (OD)](/wiki/Beatmapping/Overall_difficulty) sowie die [Gesundheit (HP)](/wiki/Gameplay/Health) erhöht werden.
+
+Die Mod Double Time wird weithin als eine der besten Mods beim Erlangen großer Mengen von [Performance-Punkten](/wiki/Performance_points) auf einfacheren Schwierigkeitsgraden in [osu!](/wiki/Game_mode/osu!) angesehen.
+
+### osu!taiko
+
+In [osu!taiko](/wiki/Game_mode/osu!taiko) wird die Trefferquote stark reduziert, sobald die Mod Double Time aktiviert wird, da osu!taiko bereits eine strikte allgemeine Schwierigkeit hat, zusammen mit den wesentlich schwieriger zu treffenden Sliderticks. Aus diesem Grund wird die Mod Double Time weithin als die härteste Mod in osu!taiko angesehen und wird nur selten benutzt.
+
+### osu!catch
+
+In [osu!catch](/wiki/Game_mode/osu!catch) gibt es keine allgemeine Schwierigkeit, die erhöht werden kann. Daher erhöht diese Mod nur die BPM und die AR. Infolgedessen ist sie nur einen 1,06-fachen Multiplikator wert (im Gegensatz zu einem 1,12-fachen Multiplikator bei jeder anderen Mod).
+
+Zwar kommen bei dieser Mod Hyperdashes häufiger vor, allerdings sind Fruits in einigen Fällen trotz der Hyperdashes fast unmöglich zu fangen.
+
+## Trivia
+
+- Wenn die Mod Double Time aktiviert ist, sind die Angaben `Length`, `BPM` und `Objects` rötlich mit anderen Werten gefärbt (siehe Abbildung unten).
```

Comment:
```suggestion
- Wenn die Mod Double Time aktiviert ist, sind die Angaben `Dauer`, `BPM` und `Objekte` rötlich mit anderen Werten gefärbt (siehe Abbildung unten).
```

![image](https://user-images.githubusercontent.com/22643729/197363556-db373adc-2078-4934-b0f5-36ce35072ae4.png)

Reply:
The article refers to a screenshot that is in English. Let me adjust the picture. I prefer screenshots in German anyway.